frontendUiApi/src/main/java/com/virtuslab/gitmachete/frontend/ui/api/table/BaseGraphTable.java

```diff
@@ -1,40 +1,32 @@
 package com.virtuslab.gitmachete.frontend.ui.api.table;
 
-import javax.swing.table.AbstractTableModel;
+import javax.swing.table.TableModel;
 
+import com.intellij.ui.SimpleTextAttributes;
 import com.intellij.ui.table.JBTable;
+import com.intellij.util.ui.JBUI;
 import org.checkerframework.checker.guieffect.qual.UI;
 import org.checkerframework.checker.guieffect.qual.UIEffect;
+import org.checkerframework.checker.nullness.qual.Nullable;
 
 public abstract class BaseGraphTable extends JBTable {
-
   @UIEffect
-  protected BaseGraphTable(AbstractTableModel model) {
+  public BaseGraphTable(TableModel model) {
```

Comment:
AFAIK it doesn't make sense to have a public (and not protected) c'tor in an abstract class (but it doesn't matter much anyway)

Reply:
Yeah, you are right. It's just default for new class template in IntelliJ